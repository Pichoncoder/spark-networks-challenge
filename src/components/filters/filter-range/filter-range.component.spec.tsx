import React from 'react'
import { render, fireEvent } from '@testing-library/react'
import TestRenderer, { create } from 'react-test-renderer';
import FilterRange from './filter-range.component';
import { shallow } from 'enzyme';

describe('<FilterRange /> spec', () => {
  it('Snapshot:renders the component', () => {
    const props ={ handleRange: () => true, text: 'Age', filter:"", value: [], min:18 , max:99 } ;

      const container = TestRenderer.create(<FilterRange {...props}></FilterRange>);
      expect(container).toMatchSnapshot();
  });
});