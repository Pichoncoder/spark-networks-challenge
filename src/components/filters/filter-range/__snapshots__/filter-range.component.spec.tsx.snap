// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterRange /> spec Snapshot:renders the component 1`] = `
<li
  className="filter-element"
>
  <label>
    Age
    <div
      className="rc-slider"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      onMouseUp={[Function]}
      onTouchStart={[Function]}
    >
      <div
        className="rc-slider-rail"
        style={Object {}}
      />
      <div
        className="rc-slider-track rc-slider-track-1"
        style={
          Object {
            "left": "0%",
            "width": "100%",
          }
        }
      />
      <div
        className="rc-slider-step"
      />
      <div
        aria-disabled={false}
        aria-valuemax={99}
        aria-valuemin={18}
        aria-valuenow={18}
        className="rc-slider-handle rc-slider-handle-1"
        onBlur={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        role="slider"
        style={
          Object {
            "left": "0%",
          }
        }
        tabIndex={0}
      />
      <div
        aria-disabled={false}
        aria-valuemax={99}
        aria-valuemin={18}
        aria-valuenow={99}
        className="rc-slider-handle rc-slider-handle-2"
        onBlur={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        role="slider"
        style={
          Object {
            "left": "100%",
          }
        }
        tabIndex={0}
      />
      <div
        className="rc-slider-mark"
      />
    </div>
    <small>
       - 
    </small>
  </label>
</li>
`;
